src/data.js

```diff
@@ -388,11 +388,16 @@ export class RepoAPI {
         })
     }
 
-    create(repository) {
-        const fullName = [repository.owner, repository.name].join("/")
+    create(username, repository) {
+        const fullName = [username, repository.name].join("/")
 
         return new Promise((resolve, reject) => {
-            if (!data.accounts.has(repository.owner)) {
+            if (!repository.name.match(/^[a-zA-Z0-9_\-]{2,20}$/)) {
+                reject(Error("Repository name must only contain alphanumeric characters"))
```

Comment:
Min and Max length is checked via the regular expression, but in case of a corresponding error not communicated to the user.